c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -4402,6 +4403,18 @@ bool process_network_io(JSContext *cx) {
   return true;
 }
 
+bool math_random(JSContext *cx, unsigned argc, Value *vp) {
+  int32_t storage;
+  int32_t *buf = &storage;
+  random_get((int32_t)buf, 8);
+  uint32_t value = std::abs(storage);
+  double newvalue = static_cast<float>(value) / std::powf(2.0, 32.0);
```

Comment:
I'd probably do this all as a `double` if the result will be used as a double.
```suggestion
  double newvalue = static_cast<double>(value) / std::pow(2.0, 32.0);
```